frontends/p4/typeChecking/typeCheckTypes.cpp

```diff
@@ -331,13 +331,13 @@ static bool checkEnumValueInitializer(const IR::Type_Bits *type, const IR::Expre
             std::string extraMsg;
             if (!type->isSigned && constant->value < low) {
                 extraMsg =
-                    str(boost::format(
```

Comment:
This one only does string interpolation. Maybe get rid of `boost::format` entirely here to reduce the footprint. Same below.